scripts/linux_share_build.sh

```diff
@@ -56,36 +58,65 @@ function build_dagmc(){
                 -DBUILD_EXE=OFF \
                 -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
                 -DCMAKE_BUILD_TYPE=Release \
+                $CMAKE_ADDITIONAL_FLAG \
                 -DCMAKE_INSTALL_PREFIX=${PLUGIN_ABS_PATH}/DAGMC
-    make -j$PROC
+                
+    
+    make -j`grep -c processor /proc/cpuinfo`
     make install
     cd ../..
-    rm -rf DAGMC/DAGMC DAGMC/bld
+    rm -rf DAGMC/DAGMC DAGCM/bld
 }
 
+
 function setup_Trelis_sdk() {
-    cd /Trelis-sdk 
-    dpkg -i Trelis-$1-Lin64.deb
+    if [ "$1" = "2020.2" ]; then
+        TRELIS_PATH="/opt/Coreform-Cubit-2020.2"
+        TRELIS_PKG="Coreform-Cubit-2020.2-Lin64.deb"
+        TRELIS_SDK_PKG="Coreform-Cubit-2020.2-Lin64-SDK.tar.gz"
+    elif [ "$1" = "17.1.0" ]; then
+        TRELIS_PATH="/opt/Trelis-17.1"
+        TRELIS_PKG="Trelis-17.1.0-Lin64.deb"
+        TRELIS_SDK_PKG="Trelis-SDK-17.1.0-Lin64.tar.gz"
+        CMAKE_ADDITIONAL_FLAG="-DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=0"
+    else
+        echo "unknown Trelis/Cubit version, use: \"17.1\" or \"2020.2\""
+    fi
+    if [ "$1" = "2020.2" ]; then
+        apt-get -y install libxt6 libglew2.1 libglu1-mesa libopengl0 ocl-icd-libopencl1 libcxsparse3 libopenblas0 libegl1 libboost-thread1.71.0 liblapack3
```

Comment:
Some of these packages are present in the GHA ubuntu images, but not the docker ubuntu images I take it?

Reply:
maybe yes, 
I did not understood why one was working and not the other...

Reply:
Hopefully the above comment about using apt will take care of this too.